### PR TITLE
clean up episodic ataxia type 1 and 2

### LINKED ISSUE #8724: review episodic ataxia classification
From Chris:
is-a assertions at this level are a red flag. Are EA1 and EA2 so different? Is EA1 really a neuropathy?

in fact there is zero support for this is-a.

Image

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
- revise definition to match current Orphanet def
- revise synonym scope to match OMIM
- add xrefs

close #8724

### CODE REVIEW

**Review by katiermullen — COMMENTED** [review]
@nicolevasilevsky I have concerns about spitting out the synonyms currently associated with EA1. This disease is a central and peripheral neuropathy that is caused by a mutation in the KCNA1 gene that results in potassium channelopathy characterized by constant myokymia and episodes of spastic contractions of the skeletal muscles of the head, arms, and legs with loss of both motor coordination and balance (ataxia).

Also, note to self, we need to obsolete: 'acetazolamide-responsive hereditary episodic ataxia'

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -439594,17 +439588,19 @@ subset: rare
 synonym: "acquired neuromyotonia" EXACT []
 synonym: "continuous muscle fiber activity syndrome" EXACT [Orphanet:84142]
 synonym: "continuous muscle fibre activity syndrome" EXACT OMO:0003005 []
-synonym: "Isaac's-Merten's syndrome" RELATED [GARD:0006793]
+synonym: "Isaac's-Merten's syndrome" EXACT [GARD:0006793]
 synonym: "Isaac-Mertens syndrome" EXACT []
-synonym: "Isaacs' syndrome" RELATED [GARD:0006793]
-synonym: "neuromyotonia" RELATED [GARD:0006793]
+synonym: "Isaacs' syndrome" EXACT [GARD:0006793]
+synonym: "Isaacs-Mertens syndrome" EXACT [https://www.ncbi.nlm.nih.gov/books/NBK25442/, OMIM:160120, Orphanet:84142]
```
[the remark this answers is not in the document]
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:439594]: I moved this synonym back to 'episodic ataxia type 1'

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -544720,6 +544721,15 @@ intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.o
 intersection_of: predisposes_towards MONDO:0014005 ! immunoglobulin-mediated membranoproliferative glomerulonephritis
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/8517" xsd:anyURI
 
+[Term]
+id: MONDO:0100594
+name: myokymia 1
```
[the remark this answers is not in the document]
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:544726]: This is tricky and I'm not totally sure how to proceed with this - but this is my understanding.

Myokynia is a phenotype (see [Myokymia HP:0002411](https://hpo.jax.org/browse/term/HP:0002411)).

In Mondo, we have MONDO:0016373 'isolated facial myokymia'- there are cases in Mondo where we have isolated cases of the phenotype and consider this a disease. I added your 2nd definition above to that term.

From what I read in the paper you linked above, it sounds like there is a hereditary form, which OMIM calls myokymia 1 (syn: hereditary myokymia and myokymia with episodic ataxia (from the paper)). I don't think it should be a child of episodic ataxia, because it sounds like that is a feature of the myokymia, not a type of episodic ataxia.

I think isolated and myokymia 1 should be siblings, not parent/child.

**Comment by nicolevasilevsky** [comment]
> Also, note to self, we need to obsolete: 'acetazolamide-responsive hereditary episodic ataxia'

I'll do this on a separate PR

File: src/ontology/mondo-edit.obo [hunk 3]
```diff
@@ -451551,14 +451550,19 @@ subset: orphanet_rare {source="Orphanet:972"}
 subset: otar {source="MONDO:OTAR"}
 subset: rare
 synonym: "continuous muscle fiber activity hereditary" EXACT [MONDO:0022857]
+synonym: "continuous muscle fiber activity, hereditary" EXACT [OMIM:160120]
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:451553]: @katiermullen do you think it's appropriate to add these synonyms here?

**Comment by nicolevasilevsky** [comment]
@katiermullen please take a look at the changes I made and let me know if this addresses all of your requests, thanks!

File: src/ontology/mondo-edit.obo (hunk 3 above)
Comment (katiermullen) [reply src/ontology/mondo-edit.obo:451553]: Yes, I thin this is okay because orphanet references this OMIM term.

File: src/ontology/mondo-edit.obo (hunk 2 above)
[the remark this answers is not in the document]
Comment (katiermullen) [reply src/ontology/mondo-edit.obo]: I agree this is best for now. We can always check with the medical experts if 'myokymia 1' should be its own class.

**Review by katiermullen — APPROVED** [review]
Thanks @nicolevasilevsky for addressing all of the comments! I reviewed and approve this PR.
